addons/project/models/project.py

```diff
@@ -1725,7 +1730,8 @@ def copy(self, default=None):
             default['recurrence_id'] = self.recurrence_id.copy().id
         if self.allow_subtasks:
             default['child_ids'] = [child.copy({'name': child.name} if has_default_name else None).id for child in self.child_ids]
-        task_copy = super(Task, self).copy(default)
+        self_with_mail_context = self.with_context(mail_auto_subscribe_no_notify=True, mail_create_nosubscribe=True, mail_notrack=True)
```

Comment:
Do we really want to remove the tracking values when we copy a task/project? 🤔 
I think it is because of you disabled the tracking on the fields with `mail_notrack=True` that the burndown chart test failed.

(the comment is also valid for the copy in `project.project`)

Reply:
Yep indeed 🙂